plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java

```diff
@@ -222,25 +235,64 @@ protected List<? extends SQLScriptElement> translateStatement(
                     }
                 }
             }
-            if (defChanged) {
-                String newQueryText = SQLFormatUtils.formatSQL(null,
+            if (dialectName.equals(DIALECT_NAME_ORACLE) && !CommonUtils.isEmpty(createTable.getIndexes())) {
```

Comment:
It is a bad idea to use if/else with dialect names. This logic must be implement inside dialect implementation.

Reply:
fixed